app/models/participant_declaration.rb

```diff
@@ -39,6 +39,7 @@ class ParticipantDeclaration < ApplicationRecord
   scope :mentor, -> { where(participant_profile_id: ParticipantProfile::Mentor.select(:id)) }
   scope :npq, -> { where(participant_profile_id: ParticipantProfile::NPQ.select(:id)) }
 
+  scope :unique_paid_payable_or_eligible, -> { where(state: %w[eligible payable paid]).unique_id }
```

Comment:
it's fine for now. but long term i don't think i'm a fan of these composite scopes. I rather just have `paid_payable_or_eligible` and later you can chain with `paid_payable_or_eligible.unique_id`

Reply:
Yeah, completely fair, I'll update this